### PR TITLE
Add another test to handleUserJoinedTeamEvent() 

### PR DESCRIPTION
**Pull request by kurtishouser**
Used sinon and proxyquire.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +42,8 @@
     "@slack/events-api": "1.0.1",
     "chai": "4.1.0",
     "mocha": "3.4.2",
+    "nock": "9.0.14",
+    "proxyquire": "^1.8.0",
```
Comment (HamidAghdaee) [inline package.json]: please don't use ^

**Comment by HamidAghdaee** [comment]
@kurtishouser looks good. Please just make the change requested. thanks

File: package.json (hunk 1 above)
Comment (kurtishouser) [reply package.json]: Removed. And I misspelled json in the commit message...

File: test/message-event-handlers.test.js [hunk 2]
```diff
@@ -34,9 +38,72 @@ describe('Slack message event handlers', () => {
...
+      image_24: user.profile.image_24,
+      image_512: user.profile.image_512,
+    };
+
     it('should be a function', () => {
       expect(handleUserJoinedTeamEvent).is.a('function');
     });
+
+    it('should call addUser() with correct arguments', () => {
+      const addUserStub = sinon.stub(userRepository, 'addUser');
```
Comment (joshwyatt) [inline test/message-event-handlers.test.js:98]: @kurtishouser look at this tag team!

**Pull request merged**